aspnetcore/blazor/performance/webassembly-event-pipe-diagnostics.md

```diff
@@ -41,10 +41,25 @@ The MSBuild properties in the following table enable profiler integration.
 
 Property | Default | Set value to&hellip; | Description
 --- | :---: | :---: | ---
-`<WasmPerfTracing>` | `false` | `true` | Controls diagnostic server tracing.
+`<WasmPerfTracing>` | `false` | `true` | Enables diagnostic server.
```

Comment:
@pavelsavara How is this property different from what `EventSourceSupport` does?

I'll suggest something more generic for now:

```suggestion
`<WasmPerfTracing>` | `false` | `true` | Enables support for WebAssembly performance tracing.
```